src/lib/expression/lqp_select_expression.hpp

```diff
@@ -0,0 +1,41 @@
+#pragma once
+
+#include <utility>
+
+#include "abstract_expression.hpp"
+#include "parameter_expression.hpp"
+
+namespace opossum {
+
+class AbstractLQPNode;
+
+/**
+ * Parameters are passed in as two vectors (`parameter_ids` and `parameter_expressions` that need to have the same
```

Comment:
We can pass the vector of expressions down to the AbstractExpression as the arguments of this expression.
Added
```
 * (Two separate vectors are used instead of a vector of pairs so `parameter_expressions` can be passed to the
 * AbstractExpression as they are the `arguments` to the LQPSelectExpression) 
```